### PR TITLE
Remove child from list if added as child

### PR DESCRIPTION
**Pull request by james-woo**

### CODE REVIEW

File: views/edit_or_add_mixin.pug [hunk 1]
```diff
@@ -43,10 +45,11 @@ mixin children_add(thisChoral)
         select#childChoral.form-control.add-child-input(name='childChoral')
           option(value="") No Children Added
           each choral in chorals
-            -if(!thisChoral || (choral.choralId != thisChoral.choralId))
+            -if((!thisChoral || (choral.choralId != thisChoral.choralId)) && (!children || !(children.find((c) => c.choralId == choral.choralId ))))
```
Comment (andreirtaylor) [inline views/edit_or_add_mixin.pug]: I don't think this is a good idea it means that if you accidentally remove a child you can't read it. Better to do this all in jQuery

Comment (james-woo) [reply views/edit_or_add_mixin.pug]: What do you mean? Give me an example?

**Comment by andreirtaylor** [comment]
A choral has a child, you go do edit, delete the child, try to re add it.

File: views/edit_or_add_mixin.pug (hunk 1 above)
Comment (james-woo) [reply views/edit_or_add_mixin.pug]: image

**Pull request merged**